### DISCUSSION TITLE
GOOGLE API KEY on Mac

### DISCUSSION THREAD

**Discussion by amitshipra** [danielmiessler/Fabric#763] [Q&A]
I have setup the Fabric on Mac and set it up with OPENAI, YOUTUBE and GOOGLE API key. 
Both OPENAI and YT works but gives error on Google. Is it compatible for Google API on Mac.

The key is setup for generative AI service and with same key, I can access Gemini Pro using Python API

Error:
_`I0000 00:00:1721755402.104173 266939917 check_gcp_environment_no_op.cc:29] ALTS: Platforms other than Linux and Windows are not supported`_

**Comment by Matchinski** [comment]
Exact same, no solution found...

**Reply by amitshipra** [reply]
From the looks of it, it seems some issue with the dependent library but expert can comment on it. Not sure if its an easy fix or involved.
